misc/examples.css

```diff
@@ -64,6 +64,10 @@ hr {
   padding-left: 5px;
 }
 
+.list-view-container .fa {
+  padding-left: 0px;
+}
+
```

Comment:
Because the example css overwrite the default span behavior for list-view-container spans just above this.  This fixes that for the specific case I had to add. @jeff-phillips-18